test/integration/test_cli_commands.py

```diff
@@ -874,3 +874,21 @@ def test_logging_with_logformat_logstash_flag(self):
                     obj={"filters":[]})
         output = sorted(result.output.splitlines(), reverse=True)[:4]
         self.assertEqual(expected, output)
+
+class TestCLIRepositoryCreate(CuratorTestCase):
+    def test_create_fs_repository(self):
+        test = clicktest.CliRunner()
+        result = test.invoke(
+                    curator.repomgrcli,
+                    [
+                        '--debug',
+                        '--logfile', os.devnull,
+                        '--host', host,
+                        '--port', str(port),
+                        'create',
+                        'fs',
+                        '--repository', self.args['repository'],
+                        '--location', '/tmp'
```

Comment:
This should probably not be hard-coded to `/tmp` because it is not OS independent.  Perhaps a tmpdir provided by Python? 

```
import tempfile
tmpdir = tempfile.gettempdir()
```